cmd/panicmonitor/go.mod

```diff
@@ -0,0 +1,8 @@
+module panicmonitor
+
+go 1.13
+
+require (
+	github.com/lujjjh/panicmonitor v0.0.0-20190221035312-374cf802ab79
```

Comment:
Since go.mod's placed at the root of the project, this dependency could be removed.